python/hail/expr/expressions/typed_expressions.py

```diff
@@ -2714,6 +2715,51 @@ def sequence_context(self, before=0, after=0):
         return hl.get_sequence(self.contig, self.position, before, after, rg)
 
 
+    @typecheck_method(dest_reference_genome=reference_genome_type,
+                      min_match=float)
+    def liftover(self, dest_reference_genome, min_match=0.95):
```

Comment:
I have a slight preference for putting this in `functions` rather than on `LocusExpression`. I think the expression methods are slightly more hidden in the docs, so it'll get more exposure in functions. But I don't really care

Reply:
I put the function on both LocusExpression and in functions. the one in functions takes both a locus and a locus interval. Are you okay with this or should we just have it in one place?

Reply:
oops, I meant to delete this or rewrite it. I think it's probably simpler to have it in once place, so if we change docs / anything about it, we don't need to maintain two things. But up to you

Reply:
I was following what we did for the fasta sequence where it is in both places. However, now that I think about it more carefully, the use case was slightly different and the function signature was different (contig, position) vs. locus. I'll delete the one on LocusExpression.